tests/dat/manifest_data_rule_env_var.yaml

```diff
@@ -0,0 +1,14 @@
+# This test file is used to test the basic Trigger-Action-Rule grammar
+packages:
+  manifest1:
+      version: 1.0
+      license: Apache-2.0
+      actions:
+        first_action:
+          function: actions/hello.js
+      triggers:
+        $trigger_name:
+      rules:
+        $rule_name:
+          trigger: $trigger_name
+          action: $action_name
```

Comment:
$action_name replacement should be tested elsewhere? trigger-rule indeed belong together here.

Reply:
hrmm, it's not the actual action name replacement. Just the ability to parameterize the action being connected in the rule.